inc/admin/admin.php

```diff
@@ -457,3 +457,118 @@ function rocket_add_imagify_api_result( $result, $action, $args ) {
 	return $result;
 }
 add_filter( 'plugins_api_result', 'rocket_add_imagify_api_result', 11, 3 );
+
+/**
+ * Gets all data to send to the analytics system
+ *
+ * @since 2.11
+ * @author Remy Perona
+ *
+ * @return array An array of data
+ */
+function rocket_analytics_data() {
+	global $wp_version, $is_nginx, $is_apache, $is_iis7, $is_IIS;
+
+	$untracked_wp_rocket_options = array(
+		'license',
+		'consumer_email',
+		'consumer_key',
+		'secret_key',
+		'secret_cache_key',
+		'minify_css_key',
+		'minify_js_key',
+		'sitemaps',
+		'cdn_zone',
+		'cdn_cnames',
+		'cloudflare_email',
+		'cloudflare_api_key',
+		'cloudflare_domain',
+		'cloudflare_zone_id',
+		'cloudflare_old_settings',
+		'analytics_enabled',
+		'wl_author',
+		'wl_author_URI',
+		'wl_description',
+		'wl_plugin_URI',
+		'wl_plugin_name',
+		'wl_plugin_slug',
+	);
+
+	$theme = wp_get_theme();
+	$data  = get_option( WP_ROCKET_SLUG );
+
+	foreach ( $untracked_wp_rocket_options as $untracked_option ) {
+		if ( isset( $data[ $untracked_option ] ) ) {
```

Comment:
A simple remark: this `if ( isset() ) {` is useless and can be removed, no php notice will be triggered.

Reply:
Oh, even better:

	$untracked = array_flip( $untracked_wp_rocket_options );
	$data      = array_diff_key( $data, $untracked );